docs/products/clickhouse/howto/integrate-kafka.rst

```diff
@@ -0,0 +1,164 @@
+
+Behind the scenes the integration between Aiven for ClickHouse and Apache Kafka services relies on `ClickHouse Kafka Engine <https://clickhouse.com/docs/en/engines/table-engines/integrations/kafka/>`_.
+
+.. note::
+
+    ClickHouse service integrations are available for Startup plans and higher.
+
+Prerequisites
+-------------
+
+You will need:
+
+* Aiven for ClickHouse service
+* Aiven for Apache Kafka service or a self-hosted Apache Kafka service
+* At least one topic in the Apache Kafka service
+
+Variables
+-------------
+
+The following variables will be used later in the code snippets:
+
+============================     ==========================================================================================================
+Variable                         Description
+============================     ==========================================================================================================
+``CLICKHOUSE_SERVICE_NAME``      Name of your Aiven for ClickHouse service.
+``KAFKA_SERVICE_NAME``           Name of the Apache Kafka service you use for the integration.
+``PROJECT``                      Name of Aiven project where your services are located.
+``CONNECTOR_TABLE_NAME``         Name of the Kafka engine virtual table that is used as a connector.
+``DATA_FORMAT``                  Input/output data format in which data is accepted into ClickHouse. See :doc:`../reference/supported-input-output-formats`
+``CONSUMER_GROUP_NAME``          Name of the consumer group. Each message is delivered once per consumer group.
+============================     ==========================================================================================================
+
+Create an integration
+----------------------
+
+You can create an integration with the help of :ref:`Aiven CLI <avn_service_integration_create>`, or through the `Aiven console <https://console.aiven.io/>`_ by following these steps:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_, choose the right project, and select your Aiven for ClickHouse service.
+#. In the *Overview* tab, scroll till the section *Service integrations* and click on **Set up integration**. A modal window with available integration options will appear.
+#. Select Apache Kafka from the list of the options.
+#. Choose your service from the list. This is also a place where you can trigger creation of a new Aiven for Apache Kafka service.
+#. Click on **Enable**.
+#. Your Apache Kafka service will be added to the list of enabled service integrations.
+#. You can now close *Service integrations* modal window.
+#. The Kafka integration table will now appear in the database.
+
+.. note::
+
+    Creation of this default configuration only creates an empty ClickHouse but does not create any table. This database is named after the Kafka service.
```

Comment:
I've rephrased it better and included extra details.
Overall, for the connection we need to create Kafka Engine table. In Aiven we do it in two steps - we first create a separate database (which technically is not necessary, but this will affect the queries customers run), and then we add a table there. I felt it is worth pointing out to understand what is happening behind the scenes.

Let me know if the clarification works better, or, maybe, we should omit the whole phrase altogether.